api/HTMLImageElement.json

```diff
@@ -822,10 +822,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": false
```

Comment:
This is also a capitalization issue. I've removed the custom IDL for this as well in https://github.com/foolip/mdn-bcd-collector/pull/807.

@vinyldarkscratch can you rename this entry too?